packages/lib-utils/src/components/details-page-header/DetailsPageHeader.tsx

```diff
@@ -7,7 +7,7 @@ import {
   Label,
   DropdownPosition,
 } from '@patternfly/react-core';
-import * as _ from 'lodash';
+import * as _ from 'lodash-es';
```

Comment:
@vojtechszocs I see in some other locations we removed the `-es` and here we are adding it. Can we come up with a consistent losdah import across all of our packages?